src/abstractdataframe/reshape.jl

```diff
@@ -12,9 +12,10 @@ column `variable_name` (`:variable` by default) a vector of `Symbol`s holding
 the name of the corresponding `measure_vars` variable,
 and with columns for each of the `id_vars`.
 
-See also [`stackdf`](@ref)) and [`meltdf`](@ref) for stacking methods that return a
-view into the original data frame. See [`unstack`](@ref) for converting from
-long to wide format.
+If `view=true` then return a stacked view of a data frame (long format).
+The result is a view because the columns are special `AbstractVectors`
+that return indexed views into the original data frame.
```

Comment:
What does "indexed" mean?

Reply:
I just copy-pasted this from the original description. I guess indexed is intended to mean that it is a custom view. But probably we can just remove "indexed". OK?

Reply:
I have removed "indexed"